packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -83,6 +83,9 @@ export enum TelemetryEvent {
 
   ClickQuickStartCard = "click-quick-start-card",
 
+  GetStartedPrerequisitesStart = "get-started-prerequisites-start",
+  GetStartedPrerequisites = "get-started-prerequisites-start",
```

Comment:
I think it is a typo. "get-started-prerequisites" ?

Reply:
Good catch!! Fixed